public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -62,17 +62,19 @@ import {
 } from './useContactPoints';
 import { ContactPointWithMetadata, getReceiverDescription, isProvisioned, ReceiverConfigWithMetadata } from './utils';
 
-enum ActiveTab {
-  ContactPoints,
-  NotificationTemplates,
+export enum ActiveTab {
+  ContactPoints = 'contact_points',
+  NotificationTemplates = 'templates',
 }
 
 const DEFAULT_PAGE_SIZE = 10;
 
 const ContactPoints = () => {
   const { selectedAlertmanager } = useAlertmanager();
-  // TODO hook up to query params
-  const [activeTab, setActiveTab] = useState<ActiveTab>(ActiveTab.ContactPoints);
+  const [queryParams, setQueryParams] = useQueryParams();
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+  const initialActiveTab = queryParams.tab as ActiveTab;
```

Comment:
If the user changes the param in the URL the value will be incorrect
Perhaps a custom typeguard returning `ActiveTab` would be better?